package.json

```diff
@@ -108,6 +106,8 @@
         "popper.js": "^1.14.6",
         "react-jsonschema-form": "^1.0.6",
```

Comment:
while you are in here - these 2 should be real dependencies
- react-jsonschema-form
- react-modal